domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/model/AcpFAQQuestion.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2017 the original author or authors.
+ *
+ * This file is part of jBB Application Project.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  You may obtain a copy of the License at
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package org.jbb.frontend.api.faq.model;
+
+/**
+ * Created by Piotr on 2017-06-07.
+ */
+public interface AcpFAQQuestion {
+
+    String getQuestion();
```

Comment:
Maybe just `getContent()`? Now you can have code like: `question.getQuestion()` - it is not clear for me